### ISSUE TITLE
Color Contrast: Featured Label

### ISSUE THREAD

**Issue by AutoSponge** [Hashnode/user-feedback#169]
Element has insufficient color contrast of 2.32 (foreground color: `#d9a036`, background color: `#ffffff`, font size: 10.5pt (14px), font weight: normal). 

Expected contrast ratio of 4.5:1, or 3:1 for font-weight 700+ or font-size 18pt.

Element location: `.featured-label`
Element source: `<p class="featured-label">Today's Pick</p>`

Suggested fix:
- Change font-color to `#bd8b2f`
- Change font-size to `19px`
- Change font-weight to `700`
- Makes it 3.03:1

Before:
<img width="223" alt="current font too small, thin, and light." src="https://user-images.githubusercontent.com/654530/77119079-199b9580-6a0c-11ea-893a-51b9b7d4e144.png">

<img width="222" alt="darker, larger, and bolder font." src="https://user-images.githubusercontent.com/654530/77119113-320bb000-6a0c-11ea-983f-9eea4f889fef.png">

**Comment by BolajiAyodeji** [comment]
Hi @AutoSponge,

Thanks for the amazing feedback. We are working on getting this fixed and we'll update you soon.
